psi4/share/psi4/plugin/dfmp2/plugin.cc.template

```diff
@@ -108,17 +107,21 @@ SharedWavefunction @plugin@(SharedWavefunction ref_wfn, Options& options)
 
     // => Setup DF Integrals <= //
 
-    std::shared_ptr<DFERI> df = DFERI::build(primary, auxiliary, options, ref_wfn);
-    df->add_pair_space("B", "ACTIVE_OCC", "ACTIVE_VIR");
-
-    df->set_memory(memory / 8L);
-    df->print_header();
-    df->compute();
-
-    std::shared_ptr<Tensor> B = df->ints()["B"];
-
-    FILE* Bf = B->file_pointer();
-    df.reset();
+    std::shared_ptr<df_helper::DF_Helper> dfh = std::shared_ptr<df_helper::DF_Helper>(new
+    df_helper::DF_Helper(primary, auxiliary));
```

Comment:
Should this be a `auto dfh (std::make_shared<df_helper::DF_Helper>(primary, auxiliary))`?

Reply:
I agree with Lori here.  I think that setting an example of how to instantiate this cleanly will mean that we get cleaner implementations when people copy/paste this example.